tests/TestCase/View/Helper/HtmlHelperTest.php

```diff
@@ -1015,6 +1015,12 @@ public function testScript(): void
             'script' => ['src' => 'js/jquery-1.3.2.js', 'defer' => 'defer', 'encoding' => 'utf-8'],
         ];
         $this->assertHtml($expected, $result);
+
+        $result = $this->Html->script('defered', ['defer']);
```

Comment:
Hmm.. I thought at some point be deprecated such options and wanted users to use the associative form only. Guess I miss remembered.